book/guide/0-prerequisites.md

```diff
@@ -9,68 +10,71 @@
 
 ## Introduction
 
-We'll delve into the practical use of the OPUS Pool SDK for key operations like staking, unstaking, minting, burning and querying details. Drawing from an [OPUS Pool SDK Example][opus-pool-example] repository, this tutorial is a hands-on approach to understanding and leveraging the SDK's capabilities.
+We’ll delve into the practical use of the OPUS Pool SDK for key operations like staking, unstaking, minting, burning and querying details. Drawing from an [OPUS Pool SDK Example][opus-pool-example] repository, this tutorial is a hands-on approach to understanding and leveraging the SDK’s capabilities.
 Please refer to the README in the repository for setup instructions and additional context.
 
 **By the end of this tutorial, you will learn how to do the following:**
 
 -   Learn about staking and vaults
 -   Fetch and display vault details such as TVL, APY, description, and balance
--   Stake to and deposit to a vault
--   Unstake from and withdraw from a vault
+-   Stake to and deposit to a vault
+-   Unstake from and withdraw from a vault
 -   Mint the liquid staking token osETH
 -   Burn osETH to unlock staked ETH
 -   Fetch and display vault transaction history
 -   Retrieve and display rewards history
--   Build a simple Staking dApp that connects to MetaMask using wagmi
+-   Build a simple Staking dApp that connects to MetaMask using wagmi
 
 ## Prerequisites for the OPUS Pool SDK Example
 
-Before we dive into the core of our tutorial, it's essential to understand the prerequisites and the foundation on which we're building. This guide is centered around the OPUS Pool SDK Example. 
+Before we dive into the core of our tutorial, it’s essential to understand the prerequisites and the foundation on which we’re building. This guide is centered around the OPUS Pool SDK Example.
+
+The repository for this example can be found [here][opus-pool-example].
 
-The repository for this example can be found [here][opus-pool-example]. 
+Please review the README in the repository for instructions on installation and how to run the project.
```

Comment:
Would be better read as "Please review the README file"

Reply:
fixed